### PR TITLE
Couple cancer and rare disease

### PR DESCRIPTION
**Pull request by hassanfa**
Description of PR here

Changed:
- Merged cancer and rare disease views

**Review:**
- [x] code approved by @mayabrandi 
- [x] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by hassanfa** [comment]
Changes:
1. Some color code changes for WGS, WES, and TGS
2. merged cancer and rare disease view into one.
3. temporarily removed qc compare page

image

File: vogue/server/templates/qc_over_time.html [hunk 1]
```diff
@@ -44,10 +49,16 @@
 
 <script>
     var data = []
-    {% if mip_dna_results.data[selcted_metric] is defined %}
-        {% for result in mip_dna_results.data[selcted_metric] %}
+    {% if qc_dna_results.data[selcted_metric] is defined %}
+        {% for result in qc_dna_results.data[selcted_metric] %}
             // in this context january is 0, february 1,..., december 11 thereby result.x-1
-            data.push({'name': {{result.name| tojson}}, 'x': {{result.x-1}}, 'y': {{result.y}}} )
+            data.push({
```
Comment (mayabrandi) [inline vogue/server/templates/qc_over_time.html:55]: I would make a series for each application. Ill make a separate branch with suggestion

**Review by mayabrandi — COMMENTED** [review]
Great Hassan. Had some comments.

File: vogue/server/endpoints/bioinfo/cancer.py [hunk 2]
```diff
@@ -1,20 +0,0 @@
-#!/usr/bin/env python
```
[the remark this answers is not in the document]
Comment (hassanfa) [reply vogue/server/endpoints/bioinfo/cancer.py:1]: We don't have disease specific track anymore. I think we can safely delete all traces of cancer and rare disease.

File: vogue/server/templates/balsamic.html [hunk 3]
```diff
@@ -1,18 +0,0 @@
-{% extends 'layout.html' %}
```
[the remark this answers is not in the document]
Comment (hassanfa) [reply vogue/server/templates/balsamic.html:1]: Same as above, keeping Vogue pipeline agnostic will make adding new workflows much easier.

File: vogue/server/templates/qc_over_time.html (hunk 1 above)
Comment (hassanfa) [reply vogue/server/templates/qc_over_time.html]: merged your PR, this should resolve it. thanks :-)

**Comment by hassanfa** [comment]
So here is the final view for now. I think it is good to go. I'll use my hackish script to upload data from housekeeper to Vogue for all samples, until `cg` can properly upload.

image

**Comment by hassanfa** [comment]
So here is the final view for now. I think it is good to go. I'll use my hackish script to upload data from housekeeper to Vogue for all samples, until `cg` can properly upload.

vogue_capture_20210401

**Review by mayabrandi — APPROVED** [review]
Nice hassan!

**Pull request merged**